doc/run_settings.rst

```diff
@@ -0,0 +1,159 @@
+RunSettings include a `run_command` parameter for local launches
+that utilize a parallel launch binary like `mpirun`, `mpiexec`, and others.
+If no `run_command` is specified, the executable will be launched locally.
+
+The local `launcher` supports the base :ref:`RunSettings API <rs-api>`
+which can be used to run executables as well as run executables
+with arbitrary launch binaries like `mpiexec`. The local launcher
+is the default launcher for all ``Experiment`` instances.
+
+In the following example, we walk through creating and launching locally a
+``Model`` entity with a ``RunSettings`` object.
+First specify the `launcher` argument as `"local"`
+when initializing the ``Experiment``:
+
+.. code-block:: python
+
+      exp = Experiment("name-of-experiment", launcher="local")
+
+Next, we initialize a simple ``Model`` entity that echos `Hello World` to stdout.
+The ``RunSettings`` base object will be created since we set `launcher="local"` when
+initializing the experiment.
+
+.. code-block:: python
+
+      run_settings = exp.create_run_settings("echo", exe_args="Hello World")
+
+Finally, create a ``Model`` using ``Experiment.create_model()``. Specify
+the ``RunSettings`` object along with a `name` for the model:
+
+.. code-block:: python
+
+      model = exp.create_model("hello_world", run_settings)
+
+Next, launch the Model using ``Experiment.start()``:
+
+.. code-block:: python
+
+      exp.start(model)
+
+In this experiment, there is no need to use the ``Experiment.stop()`` function
+since there is no launched orchestrator.
+
+===
+HPC
+===
+To configure an entity for launch on an HPC, SmartSim offers ``RunSettings`` child classes.
+If an allocation is specified, the instance receiving these run parameters will launch on that allocation.
+Each WLM `launcher` supports different ``RunSettings`` child classes as shown below:
+
+1. The Slurm `launcher` supports:
+   - :ref:`SrunSettings<srun_api>`
+   - :ref:`OrterunSettings<openmpi_orte_api>`
+   - :ref:`MpirunSettings<openmpi_run_api>`
+   - :ref:`MpiexecSettings<openmpi_exec_api>`
+2. The PBSpro `launcher` supports:
+   - :ref:`AprunSettings<aprun_api>`
+   - :ref:`OrterunSettings<openmpi_orte_api>`
+   - :ref:`MpirunSettings<openmpi_run_api>`
+   - :ref:`MpiexecSettings<openmpi_exec_api>`
+3. The Cobalt `launcher` supports:
+   - :ref:`AprunSettings<aprun_api>`
+   - :ref:`OrterunSettings<openmpi_orte_api>`
+   - :ref:`MpirunSettings<openmpi_run_api>`
+   - :ref:`MpiexecSettings<openmpi_exec_api>`
+4. The LSF `launcher` supports:
+   - :ref:`JsrunSettings<jsrun_api>`
+   - :ref:`MpirunSettings<openmpi_run_api>`
+   - :ref:`MpiexecSettings<openmpi_exec_api>`
+
+
+In the following example, we walk through creating and launching a
+``Model`` entity with a slurm ``SrunSettings`` object.
+Begin by initializing an ``Experiment`` object
+and specifying the Slurm WLM to the `launcher` argument:
+
+.. code-block:: python
+
+      exp = Experiment("name-of-experiment", launcher="local")
```

Comment:
- We say we're going to use srun settings but the launcher is local
- where is the allocation specified. the intro text states that it must be present

Reply:
maybe a .. note:: stating the env var lookup behavior w/WLMs and an example passing it to the run settings?

Reply:
I am adding this information into a note!